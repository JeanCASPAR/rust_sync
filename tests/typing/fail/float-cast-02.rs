use rust_sync::sync;

sync! {
    node oui() = (a)
    where
        a : float = 3.0 as float;
}

fn main() {}
