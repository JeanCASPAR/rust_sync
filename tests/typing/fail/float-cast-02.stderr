error: type mismatch: `float` is not castable to float
 --> tests/typing/fail/float-cast-02.rs:6:21
  |
6 |         a : float = 3.0 as float;
  |                     ^^^^^^^^^^^^
